brainscore_language/submission/endpoints.py

```diff
@@ -94,12 +113,15 @@ def parse_args() -> argparse.Namespace:
                         help='ID of submitting user in the postgres DB')
     parser.add_argument('--author_email', type=str, nargs='?', default=None,
                         help='email associated with PR author GitHub username')
-    parser.add_argument('--specified_only', type=bool, nargs='?', default=False,
+    parser.add_argument('--specified_only', type=lambda x: bool(strtobool(x)), nargs='?', default=False,
```

Comment:
why make this a lambda instead of bool?

Reply:
`bool("False")` evaluates to `True`, so when a user specifies `specified_only=$SPECIFIED_ONLY` where `SPECIFIED_ONLY=False`, it evaluates to True.

This could also be changed to `--store_true` I suppose.

Reply:
I'm just confused why this is the `type`, isn't what you're describing an `action` in argparse?

Reply:
You're correct that argparse has the `store_true` and `store_false` actions for boolean flags, but in this case the user specifies explicitly specified_only="True" or specified_only="False", both of which evaluate to True when type=bool regardless of the action, so the `action`s do not quite work.

The way to resolve this is by either defining type as a custom callable (which is consistent with argparse's definition of [type](https://docs.python.org/3/library/argparse.html#argparse-type)), or by using a `store_true` action and changing the Jenkins run script accordingly. I think this is more modular and backward-compatible. What do you think?

Reply:
the argparse docs aren't great on this, but I thought the way to run the command would be `program.py --specified_only` without any `key=value` pair. I would personally rather go with the argparse standards

Reply:
Should be resolved in [b5ec60b](https://github.com/brain-score/language/pull/204/commits/b5ec60b6ce95ba70cebf1d470cabf0f4048463c3).